vs/dosbox-x.vcxproj

```diff
@@ -726,6 +732,7 @@ for /d %%i in ("$(SolutionDir)\..\contrib\translations\*") do copy "%%i\*.lng" "
       <WarningLevel>Level3</WarningLevel>
       <EnableEnhancedInstructionSet>StreamingSIMDExtensions</EnableEnhancedInstructionSet>
       <MultiProcessorCompilation>true</MultiProcessorCompilation>
+      <DebugInformationFormat>EditAndContinue</DebugInformationFormat>

```

Comment:
Shouldn't use `EditAndContinue` on **Release** builds since it conflicts with
```
<WholeProgramOptimization>true</WholeProgramOptimization>
```
There are couple of more places as well.

Reply:
Reverted in #5112.